### PR TITLE
[SYSSETUP][USETUP] [NLS] Update pt-PT translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[SYSSETUP] Update pt-PT translation
[USETUP] Fix typos to pt-PT translation
[NLS] Change weekdays name to capital letter

### CODE REVIEW

File: dll/win32/syssetup/lang/pt-PT.rc [hunk 1]
```diff
@@ -175,8 +175,8 @@ BEGIN
     IDS_COMPUTERSUBTITLE "Deve fornecer um nome e palavra-passe de administrador para o seu computador."
     IDS_LOCALETITLE "Configurações Regionais"
     IDS_LOCALESUBTITLE "Pode personalizar o ReactOS para diferentes regiões e línguas."
-    IDS_LOCALETEXT "The Standards and formats are set to %s, and the location is set to %s."
-    IDS_LAYOUTTEXT "Your default text input language and method is: %s keyboard layout"
+    IDS_LOCALETEXT "Os padrões e formatos são definidos como %s, e o local é definido como %s."
```
Comment (HBelusca) [inline dll/win32/syssetup/lang/pt-PT.rc:178]: Remove trailing space:
```suggestion
    IDS_LOCALETEXT "Os padrões e formatos são definidos como %s, e o local é definido como %s."
```

File: base/setup/usetup/lang/pt-PT.h [hunk 2]
```diff
@@ -220,7 +220,7 @@ static MUI_ENTRY ptPTIntroPageEntries[] =
     {
         6,
         15,
-        "Fa\207a c\242pia de segur\306n\207 dos seus dados ou teste num computador",
+        "Fa\207a c\242pia de seguran\207a dos seus dados ou teste num computador",
```
Comment (HBelusca) [inline base/setup/usetup/lang/pt-PT.h:223]: What's the corrected new word?

File: dll/win32/syssetup/lang/pt-PT.rc [hunk 3]
```diff
@@ -202,7 +202,7 @@ BEGIN
      *   samsrv.dll:IDS_USER_ADMINISTRATOR_NAME synchronized.
      *   Also check the IDD_COMPUTERPAGE dialog.
      */
-    IDS_ADMINISTRATOR_NAME "Administrator"
+    IDS_ADMINISTRATOR_NAME "Administrador"
```
Comment (HBelusca) [inline dll/win32/syssetup/lang/pt-PT.rc:205]: What's the comment just one line above about?

File: base/setup/usetup/lang/pt-PT.h (hunk 2 above)
Comment (zecarlos1957) [reply base/setup/usetup/lang/pt-PT.h:223]: segurança

File: dll/win32/syssetup/lang/pt-PT.rc (hunk 3 above)
Comment (zecarlos1957) [reply dll/win32/syssetup/lang/pt-PT.rc:205]: OPPS!

**Review by HBelusca — APPROVED** [review]

**Pull request merged**